R/tables.R

```diff
@@ -0,0 +1,60 @@
+#' Top 10 Table Summaries
+#'
+#' @import gt
+
+mpx_table_cases <- function(){
+  df <- mpx_case_refresh(mpx_linelist_path, spo_con, include_endemic = FALSE)
```

Comment:
This won't work in a package, since `mpx_linelist_path` and `spo_con` should be declared in the function itself, and mpx_case_refresh isn't defined in the package either.

How about we just let `mpx_table_cases()` accept a `df` argument and let the data load step be separate?